### ISSUE TITLE
accuracy

### ISSUE THREAD

**Issue by Qjrun** [bojone/vae#4]
hello！I wonder how to  calculate the accuracy rate in cvae_keras.py ? How to output the accuracy rate every epoch ?

**Comment by bojone** [comment]
for example, if the real label of most examples in cluster id=1 is 2, we say the cluster id 1 is corresponding to label 2.

with this mapping, we can map the all cluster ids to real labels, and calculate its accuracy.
